app/assets/src/components/views/report/ReportTable/constants.js

```diff
@@ -0,0 +1,50 @@
+const DOC_LINK =
+  "https://github.com/chanzuckerberg/idseq-dag/wiki/IDseq-Pipeline-Stage-%233:-Reporting-and-Visualization#reports";
```

Comment:
just as a reminder, we'll need to make sure these wiki page titles never change or get updated in sync